Issue
  { issueClosedAt = Just 2016 (-07) (-05) 23 : 36 : 40 UTC
  , issueUpdatedAt = 2016 (-07) (-05) 23 : 36 : 44 UTC
  , issueEventsUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/5914/events"
  , issueHtmlUrl =
      Just "https://github.com/rethinkdb/rethinkdb/issues/5914"
  , issueClosedBy = Nothing
  , issueLabels = []
  , issueNumber = 5914
  , issueAssignee = Nothing
  , issueUser =
      SimpleUser
        { simpleUserId = Id 7917
        , simpleUserLogin = N "bsharpe"
        , simpleUserAvatarUrl =
            "https://avatars.githubusercontent.com/u/7917?v=3"
        , simpleUserUrl = "https://api.github.com/users/bsharpe"
        , simpleUserType = OwnerUser
        }
  , issueTitle = "different exit with errors"
  , issuePullRequest = Nothing
  , issueUrl =
      "https://api.github.com/repos/rethinkdb/rethinkdb/issues/5914"
  , issueCreatedAt = 2016 (-07) (-05) 00 : 23 : 33 UTC
  , issueBody =
      Just
        "```\n2016-07-05T00:16:32.953895181 14746.698036s error: Error in src/serializer/log/log_serializer.cc at line 702:\n2016-07-05T00:16:32.954675442 14746.698812s error: Guarantee failed: [erase_it != offset_tokens.end()] We probably tried unregistering the same token twice.\n2016-07-05T00:16:32.954700286 14746.698837s error: Backtrace:\n2016-07-05T00:17:06.135389732 14779.879545s error: Tue Jul  5 00:16:32 2016\\n\\n1 [0xabb110]: backtrace_t::backtrace_t() at ??:?\\n2 [0xabb4a3]: format_backtrace(bool) at ??:?\\n3 [0xd3e78d]: report_fatal_error(char const*, int, char const*, ...) at ??:?\\n4 [0xa92292]: log_serializer_t::unregister_block_token(ls_block_token_pointee_t*) at ??:?\\n5 [0xa923e9]: /usr/bin/rethinkdb() [0xa923e9] at 0xa923e9 ()\\n6 [0x9de2ba]: linux_message_hub_t::on_event(int) at ??:?\\n7 [0x9db06e]: epoll_event_queue_t::run() at ??:?\\n8 [0x9d911a]: linux_thread_pool_t::start_thread(void*) at ??:?\\n9 [0x7f05d81ef184]: /lib/x86_64-linux-gnu/libpthread.so.0(+0x8184) [0x7f05d81ef184] at 0x7f05d81ef184 (/lib/x86_64-linux-gnu/libpthread.so.0)\\n10 [0x7f05d7f1c37d]: clone+0x6d at 0x7f05d7f1c37d (/lib/x86_64-linux-gnu/libc.so.6)\n2016-07-05T00:17:06.135558400 14779.879695s error: Exiting.\n```\n\n2 nodes - doing 2 shards/2 replicas  -- 33 tables -- 3 proxies connected to nodes\n"
  , issueState = "closed"
  , issueId = Id 163753657
  , issueComments = 1
  , issueMilestone =
      Just
        Milestone
          { milestoneCreator =
              SimpleUser
                { simpleUserId = Id 706854
                , simpleUserLogin = N "AtnNn"
                , simpleUserAvatarUrl =
                    "https://avatars.githubusercontent.com/u/706854?v=3"
                , simpleUserUrl = "https://api.github.com/users/AtnNn"
                , simpleUserType = OwnerUser
                }
          , milestoneDueOn = Nothing
          , milestoneOpenIssues = 1
          , milestoneNumber = 17
          , milestoneClosedIssues = 595
          , milestoneDescription =
              Just
                "The scope of this issue is covered by another issue. The closing comment should link to the other issue."
          , milestoneTitle = "duplicate"
          , milestoneUrl =
              "https://api.github.com/repos/rethinkdb/rethinkdb/milestones/17"
          , milestoneCreatedAt = 2013 (-03) (-29) 20 : 23 : 12 UTC
          , milestoneState = "closed"
          }
  }